ocaml/xapi/xapi_vm.ml

```diff
@@ -1659,6 +1659,15 @@ let set_domain_type ~__context ~self ~value =
   Db.VM.set_HVM_boot_policy ~__context ~self
     ~value:(derive_hvm_boot_policy ~domain_type:value)
 
+let set_blocked_operations ~__context ~self ~value =
+  Db.VM.set_blocked_operations ~__context ~self ~value
```

Comment:
No need for `debug "%s" __FUNCTION__` here?

Reply:
Removed them; the logs in message forwarding should be enough.